open-company/open-company-web
# Foc hover fixes

Card: https://trello.com/c/OonUscAM

To test:
- hover a read item on FoC
- [x] do you see both top and bottom border disappear into the bg color of the hovered tile?
- now add 2 must see items with another user
- hover one of them
- [x] do you see a box shadow around the hovered tile?
- [ ] respect design?
